src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -514,13 +514,36 @@ public void appendColumnsToTable(String tableName, Map<String, ColumnInfos> colu
    * Alter iceberg table to add columns according to a map from columnNames to their types
    *
    * @param tableName the name of the table
-   * @param columnInfosMap the mapping from the columnNames to their infos
+   * @param addColumnsQuery
+   * @param alterSetDataTypeQuery
    */
   @Override
   public void appendColumnsToIcebergTable(
-      String tableName, Map<String, ColumnInfos> columnInfosMap) {
+      String tableName, String addColumnsQuery, String alterSetDataTypeQuery) {
```

Comment:
Wdyt about using `Optional<String>`? I think it's more explicit than passing an empty String.

Reply:
Yes this is not good... I will change it.

Reply:
[refactor the way we execute query, plus format.sh](https://github.com/snowflakedb/snowflake-kafka-connector/pull/1007/commits/da4bbf93dc186ad278eeb12b16695bf20e0de26b)